docs/DeveloperGuide.adoc

```diff
@@ -602,47 +599,47 @@ image::AddStudentSequenceDiagram.png[]
 
-2. The TATrackerParser sees that the command is of type student and passes the
-command to the StudentCommandParser.
+2. The `TaTrackerParser` sees that the command is of type *student* and passes the
+command to the `StudentCommandParse`r.
 
-3. The StudentCommandParser sees that the command is of type add and passes the
-arguments to the AddStudentCommandParser.
+3. The `StudentCommandParser` sees that the command is of type *add* and passes the
+arguments to the `AddStudentCommandParser`.
 
-4. The AddStudentCommandParser creates a Student with the given parameters.
+4. The `AddStudentCommandParser` creates a *Student* with the given parameters.
 
-5. The AddStudentCommandParser then creates an AddStudentCommand object and passes
-it the created module. The parser then returns the AddStudentCommand
+5. The `AddStudentCommandParser` then creates an `AddStudentCommand` object and passes
+it the created module. The parser then returns the `AddStudentCommand`.
 
-6. LogicManager calls AddStudentCommand's execute method. The AddStudentCommand object
+6. `LogicManager` calls `AddStudentCommand` 's execute method. The `AddStudentCommand` object
 checks whether a student with the given matric number already exists in TA-Tracker.
 If it does, a command exception is thrown saying that a student with the matric number
 already exists in the TA-Tracker.
 
 7. If no such student exists, the student is added to the TA-Tracker.
 
 The command used to delete a student has been implemented in a similar way. The main
-difference is that the DeleteStudentCommand checks whether an object with the given
+difference is that the `DeleteStudentCommand` checks whether an object with the given
 matric number exists in the TA-Tracker. If no such student exists, a command exception
 is thrown saying that a student with the given matric number doesn't exist. If it does
 exist, the student is removed from the TA-Tracker.
 
-Note: This diagram assumes that a student with the matric number 'A0181234G' exists
+Note: This diagram assumes that a student with the matric number `A0181234G` exists
 in the TA-Tracker.
 
 .Student Delete - Sequence Diagram
 image::DeleteStudentSequenceDiagram.png[]
 
-1. `LogicManager` uses the `TATrackerParser` to first parse the user command.
+1. `LogicManager` uses the `TaTrackerParser` to first parse the user command.
 
-2. The `TATrackerParser` sees that the command is of type student and passes the
+2. The `TaTrackerParser` sees that the command is of type *student* and passes the
 command to the `StudentCommandParser`.
 
-3. The `StudentCommandParser` sees that the command is of type delete and passes the
+3. The `StudentCommandParser` sees that the command is of type *delete* and passes the
 arguments to the `DeleteStudentCommandParser`.
 
-4. The `DeleteStudentCommandParser` creates a Student with the given matric number
+4. The `DeleteStudentCommandParser` creates a *Student* with the given matric number
```

Comment:
Could you help me mention the matric number here at the end of the sentence please?

EG "... with the given matric number A0181234G"

Reply:
okie!